angular/src/lib/select/examples/select-default.component.ts

```diff
@@ -5,25 +5,43 @@ import { Component, OnInit } from '@angular/core';
   template: `
     <div class="medium-8 columns">
       <md-select
-        defaultValue="Select an option"
-        (select)="onSelect($event)"
-        buttonClass="my-custom-button-class"
-        [ngClass]="'angular-custom-class'"
-      >
-        <div md-select-option class="custom-class-prop" label="first option"></div>
-        <div md-select-option label="second option"></div>
+        [options]="people"
+        [(ngModel)]="person"
+        (handleChange)="onChange($event)"
+        optionLabel="name"
+        placeHolder="Select an option"
+        buttonClass="custom-button-class"
+        [buttonStyle]="{width: '80%'}"
+        overlayClass="testOverlayClass" >
       </md-select>
-  </div>
+    </div>
+
+  <p> ngModel: {{person ? person.name : 'none'}} <p>
   `,
   styles: []
 })
-export class ExampleSelectDefaultComponent implements OnInit {
+export class SelectDefaultComponent {
```

Comment:
Why did this get changed from our convention?

Reply:
This was the old convention naming from the old select. Removed 'Example'